e2e/cypress/integration/messaging/post_options_menu_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [number] indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Mobile view Post menu options button in RHS', () => {
```

Comment:
Please description as we're using this for test reporting.
```suggestion
describe('Messaging', () => {
```